core/src/main/java/io/questdb/MessageBusImpl.java

```diff
@@ -115,6 +119,11 @@ public MessageBusImpl(@NotNull CairoConfiguration configuration) {
         this.o3PurgePubSeq = new MPSequence(this.o3PurgeQueue.getCapacity());
         this.o3PurgeSubSeq = new MCSequence(this.o3PurgeQueue.getCapacity());
         this.o3PurgePubSeq.then(this.o3PurgeSubSeq).then(this.o3PurgePubSeq);
+
+        this.latestByQueue = new RingQueue<>(LatestByTask::new, 32); //FIXME: add configuration
```

Comment:
Intellij supports "todo", in that it highlights files that have to-do items in them.  Please use `todo` instead of `fixme`

Having said that - we should perhaps have configuration for the queue size